src/main/java/seedu/entry/Income.java

```diff
@@ -1,10 +1,20 @@
 package seedu.entry;
 
 public class Income extends Entry {
+    private static int size = 0;
 
     public Income(String description, double value) {
         this.description = description;
         this.value = value;
+        size++;
```

Comment:
https://github.com/AY2122S1-CS2113T-T12-3/tp/pull/49/files#r727849300 Same for this segment as well